test/src/parsers/HTMLlib.js

```diff
@@ -0,0 +1,35 @@
+describe('HTML Lib', function() {
+  var lib;
+
+  beforeAll(function() {
```

Comment:
This is extraneous. We should be able to just have `beforeAll(function() { lib = foam.parsers.html' });`.